README.md

```diff
@@ -425,7 +425,8 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [Youtube](https://youtu.be/ST6Z-3SlNS4), [Code](https://github.com/anindya2001/GSCEventMOD).
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
-
+- [Gao, et at., FPGA, 2022](#Gao22FPGA),  
```

Comment:
Is this a secondary entry? Where is the main one?

Reply:
Under the event processor and platform

Reply:
ok. I lost track of it for a split second.

Reply:
Thanks